src/Driver/diagnostics_configs.jl

```diff
@@ -16,26 +16,67 @@ mutable struct DiagnosticsConfiguration
     ) where {DG <: DiagnosticsGroup} = new(groups)
 end
 
+function InterpolationConfiguration(
+    ::StackedBrickTopology,
+    driver_config,
+    boundaries,
+    axes,
+)
+    grid = driver_config.grid
+    return InterpolationBrick(grid, boundaries, axes[1], axes[2], axes[3])
+end
+
+function InterpolationConfiguration(
+    ::StackedCubedSphereTopology,
+    driver_config,
+    axes;
+    nr_toler = nothing,
+)
+    FT = eltype(driver_config.grid)
+    param_set = driver_config.bl.param_set
+    grid = driver_config.grid
+    info = driver_config.config_info
+
+    _planet_radius::FT = planet_radius(param_set)
+    vert_range = grid1d(
+        _planet_radius,
+        FT(_planet_radius + info.domain_height),
+        nelem = info.nelem_vert,
+    )
+
+    return InterpolationCubedSphere(
+        grid,
+        collect(vert_range),
+        info.nelem_horz,
+        axes[1],
+        axes[2],
+        axes[3];
+        nr_toler = nr_toler,
+    )
+end
+
 """
     InterpolationConfiguration(
         driver_config::DriverConfiguration,
         boundaries::Array,
-        resolution::Tuple,
+        resolution = nothing,
+        axes = nothing;
     )
 
 Creates an `InterpolationTopology` (either an `InterpolationBrick` or an
-`InterpolationCubedSphere`) to be used with a `DiagnosticsGroup`. The axes
-are set up based on `boundaries` and `resolution`.
+`InterpolationCubedSphere`) to be used with a `DiagnosticsGroup`. Either
+`resolution` is specified, in which case the axes are set up with
+equi-distant points, or the `axes` may be specified directly (in
+lat/lon/lvl or x/y/z order).
 """
 function InterpolationConfiguration(
     driver_config::DriverConfiguration,
     boundaries::Array,
-    resolution::Tuple,
+    resolution = nothing;
+    axes = nothing,
```

Comment:
Should we add the nr_toler keyword to this function as well?

Reply:
We could. But, it doesn't apply to `InterpolationBrick`, and it's a temporary solution so I thought the simplest approach would be best.

Reply:
ok, thanks! So is the change in solid_body_rotation_fvm.jl the right way to use the tolerance?

Reply:
Yes, that's what I had in mind. :)